### ISSUE TITLE
Main OutlineEffect got a bug fix for EraseRenderer

### ISSUE THREAD

**Issue by brettmjohnson** [JimmyCushnie/Outline-Effect-but-its-faster#2]
See line 194 of OutlineEffect.cs: https://github.com/cakeslice/Outline-Effect/commit/781625ee21cd46d646fd8e6b4975398d294f0edf

**Comment by JimmyCushnie** [comment]
Sorry, I didn't see this earlier!

Is it just line 194 that matters?

**Comment by brettmjohnson** [comment]
Looks like there are some stuff commented out too around lines 338 and 351. Hard to tell if that's all with all the formatting changes.

**Comment by JimmyCushnie** [comment]
Yeah, merging the fix will be tricky. If someone makes a PR for it I'd be happy to merge, but frankly I'm just too busy with other things to do it myself.

**Comment by dosomder** [comment]
I experienced this bug. Line 194 was enough to fix it.
